gradle-plugins/src/main/java/com/linkedin/pegasus/gradle/PegasusPlugin.java

```diff
@@ -2296,11 +2296,18 @@ private static void configureDataSchemaResourcesRoot(Project project, SourceSet
       sourceDirectorySet.srcDir(dataSchemaPath);
       project.getLogger().info("Adding resource root '{}'", dataSchemaPath);
 
-      // Exclude the data schema directory from being copied into the default Jar task
+      final String extensionsSchemaPath = getExtensionSchemaPath(project, sourceSet);
+      final File extensionsSchemaRoot = project.file(extensionsSchemaPath);
+      sourceDirectorySet.srcDir(extensionsSchemaPath);
+      project.getLogger().info("Adding resource root '{}'", extensionsSchemaPath);
+
+      // Exclude the data schema and extensions schema directory from being copied into the default Jar task
       sourceDirectorySet.getFilter().exclude(fileTreeElement -> {
         final File file = fileTreeElement.getFile();
         // Traversal starts with the children of a resource root, so checking the direct parent is sufficient
-        final boolean exclude = dataSchemaRoot.equals(file.getParentFile());
+        final boolean excludeDataSchema = dataSchemaRoot.equals(file.getParentFile());
```

Comment:
nit: these variable names are misleading. It seems to indicate whether data schema/extension dirs will be excluded, but obviously we always want to do that. I would change to something like `underDataSchemaRoot` and `underExtensionsRoot`, which indicates whether this file is the child of the data schema/extension dirs.

Reply:
Updated.